source/test/TestApp/TestRunner.cpp

```diff
@@ -85,6 +85,7 @@ namespace PlayFabUnit
             test->endTime = PlayFab::GetMilliTicks();
             test->testCase->TearDown(*test);
             test->activeState = TestActiveState::COMPLETE;
+            // printf("\n%s\n", suiteTestSummary.c_str()); // If we're debugging EventTests, it's nice to see each test as it happens...
```

Comment:
might be more useful to set a #define for this

Reply:
I don't really want to "engineer a solution" for a temporary situation.